src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -140,6 +140,20 @@ public async Task SpecializeAsync()
             _logger.LogDebug("Completed language worker channel specialization");
         }
 
+        public async Task WorkerWarmup()
+        {
+            _logger.LogInformation("Starting Worker warmup");
+            _workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+
+            IRpcWorkerChannel rpcWorkerChannel = await GetChannelAsync(_workerRuntime);
+
+            if (_workerRuntime != null && rpcWorkerChannel != null)
+            {
+                await rpcWorkerChannel.SendWorkerWarmupRequest();
+            }
+            _logger.LogDebug("Completed Worker warmup");
```

Comment:
Do we still want to log 'Completed worker warmup' even if we didn't call SendWorkerWarmupRequest?